src/Microsoft.DotNet.BuildTools.TestSuite/Microsoft.DotNet.BuildTools.TestSuite.depproj

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <OutputType>Library</OutputType>
+    <NuGetTargetMoniker>.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <NuGetRuntimeIdentifier>any</NuGetRuntimeIdentifier>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="project.json" />
```

Comment:
Let's add a comment somewhere about the purpose of this project and why we're referencing xunit packages and redisting them.